packages/utils/src/index.ts

```diff
@@ -399,95 +401,70 @@ export const prepPopulateFields = (
   return memo.prepPopulateFields(populate)
 }
 
-/**
- * Not pretty - but working for now
- */
 export const patchFileUrlForCF = curry(
```

Comment:
Rewritten this method completely to work recursively. End solution was actually cleaner than the previous, so much more happy about it now. It can still be further optimized to only try and patch file URL's for explicitly set `populate` fields.